tests/shell/shell_recorder/script.dat

```diff
@@ -0,0 +1,13 @@
+Storage: dump
+Mountpoint: system/testmount/test
+File: /tmp/test.dump
+MountArgs:
```

Comment:
i was actually planing on doing this, but i couldn't find a way without using eval(1) yet.